src/main/scala/com/fulcrumgenomics/vcf/api/VcfConversions.scala

```diff
@@ -24,19 +24,17 @@
 
 package com.fulcrumgenomics.vcf.api
 
-import java.util
-import java.util.{List => JavaList}
-
 import com.fulcrumgenomics.FgBioDef._
 import com.fulcrumgenomics.fasta.SequenceMetadata
 import com.fulcrumgenomics.vcf.api.Allele.NoCallAllele
 import com.fulcrumgenomics.vcf.api.VcfCount.Fixed
 import htsjdk.variant.variantcontext.{GenotypeBuilder, VariantContext, VariantContextBuilder, Allele => JavaAllele}
 import htsjdk.variant.vcf._
 
+import java.util
+import java.util.{List => JavaList}
```

Comment:
Can these go away again now?

Reply:
they just got moved, keeping them for now so I can merge the PR